docs/contact.md

```diff
@@ -22,7 +22,7 @@ If you have a question about CSIRO's eReefs Research, you can use [the contact f
 
 ### Contact the AIMS eReefs Research Team:
 
-If you have a question about eReefs Research produced by the team at the Australian Institute of Marine Science (AIMS), please refer to the [contact information on the main AIMS website](https://www.aims.gov.au/docs/about/contacts.html).
+If you have a question about eReefs Research produced by the team at the Australian Institute of Marine Science (AIMS), please contact the AIMS eReefs team at [e&#8209;atlas+ereefs@aims.gov.au](mailto:e-atlas+ereefs@aims.gov.au).
```

Comment:
Why the HTML entity for the hyphen in the email address here?  Is that on purpose?

And just to check: are you sure you want to publish a plain email address here rather than link to a form which has a CAPTCHA or similar on it?   I am happy to approve the PR with this, but just be prepared to filter a certain amount of spam...

Reply:
I had an ordinary hyphen previously, but when I previewed it, it was breaking the word onto a new line at that point, so I opted to go with the non-breaking hyphen.

I had the same thoughts about the plain email address, but Eric was not too worried. Apparently that email address is plastered all over the internet at this point.